packages/utilities/README.md

```diff
@@ -178,7 +183,11 @@ class RateLimitExample {
 ```ts
 @SimpleCommand("hello")
 @Guard(NotBot)
-hello({ message }: SimpleCommandMessage): void {
+hello({ message }
+:
+SimpleCommandMessage
+):
+void {
```

Comment:
👀

Reply:
yeh, for some reason my formatting went mental and broke everything